truss/templates/trtllm-briton/src/engine.py

```diff
@@ -126,26 +156,15 @@ def load(self):
         briton_env = os.environ.copy()
         if self._hf_token is not None:
             briton_env["HF_ACCESS_TOKEN"] = self._hf_token
-        if self._tp_count is None or self._tp_count == 1:
-            self._briton_process = subprocess.Popen(
-                ["Briton", "--config", str(config_pbtxt_path)], env=briton_env
-            )
-        else:
-            self._briton_process = subprocess.Popen(
-                [
-                    "mpirun",
-                    "--allow-run-as-root",
-                    "-n",
-                    f"{self._tp_count}",
-                    "Briton",
-                    "--config",
-                    str(config_pbtxt_path),
-                ],
-                env=briton_env,
-            )
+
+        self._briton_process = multiprocessing.Process(
+            target=start_briton, args=(config_pbtxt_path, briton_env, self._tp_count)
+        )
+        self._briton_process.start()
+
         while is_port_available(BRITON_PORT):
             print("Waiting for Briton to start")
-            time.sleep(1)
+            time.sleep(10)
```

Comment:
Curios why increase this. Could this slow down startup for smaller models?

Reply:
Reverted this for now; we got a user feedback request to log the message less frequently. I can make a todo to poll at the same interval but print less frequently.